spec/features/apis/user_sees_api_show_page_spec.rb

```diff
@@ -1,29 +1,27 @@
-require "rails_helper"
+require 'rails_helper'
 
-# [x] I want to visit an API show page so I can review it
+feature 'user can view an API show page' do
+  let!(:user)  { FactoryGirl.create(:user, id: 1) }
+  let!(:api1)  { FactoryGirl.create(:api, user_id: 1) }
+  let!(:api2)  { FactoryGirl.create(:api, user_id: 1, paid: true) }
```

Comment:
Search for every instance of `user_id` in your codebase and change it to reflect proper convention.

```
let!(user) { FactoryGirl.create(:user) }
let!(:api1) { FactoryGirl.create(:api, user: user) }
```